st2tests/integration/mistral/test_filters.py

```diff
@@ -0,0 +1,373 @@
+                                           parameters={"input_str": REGEX_SEARCH_STR})
+        execution = self._wait_for_completion(execution)
+        self._assert_success(execution, num_tasks=1)
+        self.assertTrue(execution.result['result_jinja'])
+        self.assertTrue(execution.result['result_yaql'])
+
+    def test_regex_nosearch(self):
+        execution = self._execute_workflow('examples.mistral-customfilters-regex_search',
+                                           parameters={"input_str": REGEX_SEARCH_STR_3})
+        execution = self._wait_for_completion(execution)
+        self._assert_success(execution, num_tasks=1)
+        self.assertFalse(execution.result['result_jinja'])
+        self.assertFalse(execution.result['result_yaql'])
+
+
+class RegexSubstringFiltersTest(base.TestWorkflowExecution):
+
+    def test_regex_substring(self):
+        execution = self._execute_workflow('examples.mistral-customfilters-regex_substring',
+                                           parameters={"input_str": REGEX_SEARCH_STR})
+        execution = self._wait_for_completion(execution)
+        self._assert_success(execution, num_tasks=1)
+        self.assertEqual(execution.result['result_jinja'], '567 Elsewhere Dr')
+        self.assertEqual(execution.result['result_yaql'], '567 Elsewhere Dr')
+        self.assertEqual(execution.result['result_jinja_index_1'], '123 Somewhere Ave')
+        self.assertEqual(execution.result['result_yaql_index_1'], '123 Somewhere Ave')
+
+
+class ToHumanTimeFromSecondsFiltersTest(base.TestWorkflowExecution):
+
+    def test_to_human_time_from_seconds(self):
+        execution = self._execute_workflow(
+            'examples.mistral-customfilters-to_human_time_from_seconds',
+            parameters={"seconds": 4587}
+        )
+        execution = self._wait_for_completion(execution)
+        self._assert_success(execution, num_tasks=1)
+        self.assertEqual(execution.result['result_jinja'], '1h16m27s')
+        self.assertEqual(execution.result['result_yaql'], '1h16m27s')
+
+
+class UseNoneFiltersTest(base.TestWorkflowExecution):
+
+    def test_use_none(self):
+        inputs = {'input_str': 'foo'}
+        execution = self._execute_workflow(
+            'examples.mistral-customfilters-use_none', parameters=inputs
+        )
+        execution = self._wait_for_completion(execution)
+        self._assert_success(execution, num_tasks=2)
+        self.assertEqual(execution.result['none_result_jinja'], '%*****__%NONE%__*****%')
+        self.assertEqual(execution.result['none_result_yaql'], '%*****__%NONE%__*****%')
+        self.assertEqual(execution.result['str_result_jinja'], 'foo')
+        self.assertEqual(execution.result['str_result_yaql'], 'foo')
+
+
+class ToComplexFiltersTest(base.TestWorkflowExecution):
+
+    def test_to_complex(self):
+        execution = self._execute_workflow(
+            'examples.mistral-customfilters-to_complex'
```

Comment:
I prefer the complex object to be pass in as input parameters instead of using examples.object_return.  It's more readable for the test.